tests/Bridge/Symfony/Messenger/DataPersisterTest.php

```diff
@@ -72,4 +72,14 @@ public function testHandle()
         $dataPersister = new DataPersister($this->prophesize(ResourceMetadataFactoryInterface::class)->reveal(), $messageBus->reveal());
         $this->assertSame('result', $dataPersister->persist($dummy));
     }
+
+    public function testSupportsPerOperation()
```

Comment:
```suggestion
    public function testSupportsPerOperation(): void
```

Reply:
since when do we do this :p

Reply:
Since PHPUnit 8? :stuck_out_tongue:

Reply:
Only for:
* `PHPUnit\Framework\TestCase::setUpBeforeClass()`,
* `PHPUnit\Framework\TestCase::setUp()`,
* `PHPUnit\Framework\TestCase::assertPreConditions()`,
* `PHPUnit\Framework\TestCase::assertPostConditions()`,
* `PHPUnit\Framework\TestCase::tearDown()`,
* `PHPUnit\Framework\TestCase::tearDownAfterClass()`,
* `PHPUnit\Framework\TestCase::onNotSuccessfulTest()`.
😄

Reply:
Sure but I think it's a good habit to always do it.